_rules/95e043.md

```diff
@@ -0,0 +1,90 @@
+---
+name: Auto-play audio has a control mechanism
+test_type: atomic
+
+description: |
+  This rule checks that each auto-play audio or video element has a control mechanism.
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+- Audio output
+
+authors:
+- Anne Thyme Nørregaard
+- Bryn Anderson
+---
+
+## Applicability
+
+This rule applies to any HTML `audio` or `video` element with a `src` file content `duation` of more than 3 seconds, is set to `autoplay` and or `loop`, and has audio output that is not `paused` or `muted`.
+
+## Expectation 1
+
+For each test target a control mechanism is provided to pause or stop the audio, or turn the audio volume off independently from the overall system volume control.
+
+## Expectation 2
+
+The control mechanism to pause or stop or turn the audio volume off, is [visible](#visible), has an [accessible name](#accessible-name) and is one of the first five focusable elements in the [document](#https://www.w3.org/TR/dom/#concept-document).
+
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility Support
+
+There are no major accessibility support issues known for this rule.
+
+## Test Cases
+
+### Passed
+
+### Passed example 1
+
+The `audio` element has a control mechanism.
+
+``` html
+  <audio src="../test-assets/moon-audio/moon-speech.mp3" autoplay="true" controls></audio>
+```
+
+### Failed
```

Comment:
Include examples with controls that are not visible or have no accessible name or are not in the first five focusable elements

Reply:
Working on it :)

@jkodu I would like to upload a CSS and JS file for a custom player. Can I just go ahead and do that https://github.com/auto-wcag/auto-wcag/tree/master/assets/css and https://github.com/auto-wcag/auto-wcag/tree/master/assets/js respectively.

Reply:
OK added test cases.